src/tests/topp/IDFileConverter_16_output.idXML

```diff
@@ -39,7 +39,7 @@
 			</PeptideHit>
 		</PeptideIdentification>
 		<PeptideIdentification score_type="InterProphet probability" higher_score_better="true" significance_threshold="0" MZ="566.77033723893" RT="0" >
-			<PeptideHit score="0.0505509" sequence=".(Glu->pyro-Glu)EAGIVDELAYA" charge="2" aa_before="K" aa_after="V" protein_refs="PH_4" >
+			<PeptideHit score="0.0505509" sequence=".(Glu-&gt;pyro-Glu)EAGIVDELAYA" charge="2" aa_before="K" aa_after="V" protein_refs="PH_4" >
```

Comment:
that was probably not a good idea to have these characters as part of the  official openms names ...